tensorflow/core/kernels/conv_ops_3d.cc

```diff
@@ -450,8 +450,8 @@ struct LaunchConvOp<GPUDevice, T> {
 #if GOOGLE_CUDA
       se::TfAllocatorAdapter tf_allocator_adapter(
           stream->parent()->platform(), ctx->device()->GetAllocator({}));
-      se::cuda::RedzoneAllocator rz_allocator(
-          stream, &tf_allocator_adapter, se::cuda::PtxCompilationOptions());
+      se::RedzoneAllocator rz_allocator(stream, &tf_allocator_adapter,
+                                        se::cuda::PtxCompilationOptions());
```

Comment:
GpuAsmOptions?

Reply:
Will get rid of `se::Cuda::PtxCompialtionOptions()` everywhere, this applies to similar cases below.